Lib/Src/error_log.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file error_log.cpp
+ */
+
+/*
+ *  Created on: Feb 12, 2022
+ *      Author: niuslar
+ */
+
+#include "error_log.h"
+
+/**
+ * @brief Constructor
+ * @param Pointer to CUartCom object
+ * @param module_name Identifier that will be added to the message
+ */
+CLog::CLog(CUartCom *error_uart, const std::string module_name)
+    : mp_error_uart(error_uart),
+      m_module_name(module_name)
+{
+}
+
+/**
+ * @brief change log level to change what is sent
```

Comment:
Worth explaining what happens to messages that are below log level. Are they lost or are they stored?